src/Settings/Settings.ts

```diff
@@ -42,6 +42,8 @@ export const Settings = {
   RemoteFileApiPort: 0,
   /** Whether to save the game when the player saves any file. */
   SaveGameOnFileSave: true,
+  /** Whether to hide dialogs for stock work competed upon augmentation install. */
+  SuppressAugWorkFinishedPopup: false,
```

Comment:
Unused setting can be removed from Settings.ts

Other than that seems good.

Reply:
Updated, thank you